dev-docs/modules/consentManagement.md

```diff
@@ -128,6 +187,9 @@ gulp build --modules=consentManagement,bidAdapter1,bidAdapter2
 
 ## Adapter Integration
 
+{: .alert.alert-info :}
+Prebid.js adapters don't need to change to support TCF v2.0 -- the consent string is passed through the same bidrequest location. The bidder's endpoint, however, will need to change to support TCF v2.0.
```

Comment:
Does this mean they don't need to change if they already support v1.1? If so it would be good to explicitly call that out.

Reply:
Ok - clarified. Thanks Jean.